mirage/scenarios/default.js

```diff
@@ -862,30 +860,646 @@ export default function(server) {
 
   server.create('milestone', 'certifiedReferred', {
     project: seedBPUserProjects[0],
-    statuscode: 'Not Started',
+    statuscode: 'Completed',
     dcpActualstartdate: moment().subtract(2, 'days'),
     displayDate: moment().subtract(2, 'days'),
-    dcpActualenddate: null,
-    displayDate2: null,
+    dcpActualenddate: moment().subtract(2, 'days'),
+    displayDate2: moment().subtract(2, 'days'),
   });
 
-  // Here, mock that the BP user's first project
-  // has reached Public Review but not yet their respective Review Period
   server.create('milestone', 'communityBoardReview', {
     project: seedBPUserProjects[0],
     statuscode: 'In Progress',
-    dcpActualstartdate: moment().subtract(1, 'days'),
-    displayDate: moment().subtract(1, 'days'),
+    dcpActualstartdate: moment().subtract(2, 'days'),
+    displayDate: moment().subtract(2, 'days'),
+    dcpActualenddate: moment().add(28, 'days'),
+    displayDate2: moment().add(28, 'days'),
```

Comment:
@hannahkates   If this first project (project[0]) is 'filed', that means it hasn't reached public review right? 

If public review hasn't started, shouldn't community board review's `dcpActualenddate` not be defined yet?